index.js

```diff
@@ -23,5 +26,24 @@ module.exports.sync = (fp, data) => {
 
 	fp = path.basename(fp) === 'package.json' ? fp : path.join(fp, 'package.json');
 
+	data = normalize(data);
+
 	writeJsonFile.sync(fp, data, opts);
 };
+
+function normalize(pkg) {
+	const npkg = {};
+	for (const key of Object.keys(pkg)) {
+		switch (key) {
```

Comment:
But that would be a little bit less performant, right? I'd prefer this faster implementation if you don't insist on creating a set